src/Pages/Signup/Signup.scss

```
@@ -0,0 +1,285 @@
+@import '../../Styles/common.scss';
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css');
+.Signup-Main {
```

```suggestion
@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css');

.Signup-Main {
```
- 구분되는 요소들은 한줄씩 띄워주세요
- 컨벤션이 팀 내에서도 전반적으로 중구난방인데 팀원들과 대화해서 하나로 통일해주세요!

---

넵 알겠습니다!